### ISSUE TITLE
Wear screen configuration on phone

### ISSUE THREAD

**Issue by nebmo** [jonasoreland/runnerup#362]
Working on two things:
1. Wear layout configuration in phone settings
2. A bit diifferent UI for intervals on wear. I've added a circular progressbar that will show the current progress of the interval. Also i'm thinking about colors to indicate if below/over target pace/hr

Posting some pictures from how it looks right now. any feedback? I know Jonas would like a preview on wear feature but I'm not sure it's that easy to implement
![image](https://cloud.githubusercontent.com/assets/4735707/12546033/642d7fd4-c347-11e5-9717-10f922e94c1a.png)
![image](https://cloud.githubusercontent.com/assets/4735707/12546049/8278a23e-c347-11e5-9ef0-d80750968b19.png)
![image](https://cloud.githubusercontent.com/assets/4735707/12546064/932cd460-c347-11e5-9f51-9e07f66d2708.png)

The new Interval screen on watch. I know colors are hideous and maybe I should try to have an arrow somewhere instead.  Red means you should speed up, blue = slow down, black = at target

![screen_black](https://cloud.githubusercontent.com/assets/4735707/12546102/c5d6aec2-c347-11e5-8078-8cbeafbe71aa.png)
![screen_blue](https://cloud.githubusercontent.com/assets/4735707/12546103/c5d7188a-c347-11e5-881e-b47b69a46ea2.png)
![screen_red](https://cloud.githubusercontent.com/assets/4735707/12546104/c5d780cc-c347-11e5-83b1-c68a63192c37.png)

**Comment by jonasoreland** [comment]
feedback:
1) i think titlespinners instead of buttons for the config activity...
wdyt ? (not a deal breaker)

2) the progressbar looks cool.
it's invisible if you don't have duration, right ?

3) preview would be cool...but is also not a deal breaker.

4) color code is a good idea for Target pace/heartrate
- i think both the red/blue that you posted are awful

**Comment by nebmo** [comment]
1) I tried with titlespinner first (or with @drawable/title_spinner) but i thought it looked worse than when using the button_blue
2) Yes it's invisible when no duration
3) If i find time I will look into it. The alternative would be have some kind of preview on phone..
4) I will try and find something better and would really appreciate some help here

**Comment by nebmo** [comment]
Well, small update. Did run some intervals to test it. Colors are not working.. I think the background needs to be black, otherwise it's hard to see the numbers.

**Comment by cmmata** [comment]
Maybe you can use colors on pace numbers instead of all background. Or in that circular progressbar.

**Comment by nebmo** [comment]
@cmmata , that is actually a pretty good suggestion. I will look into that and see if I can make it work. Do you think speed up / slow down should have same or different colors?

**Comment by cmmata** [comment]
I think it would be better in different colors, so you can quickly see if you are going too fast or too slow without having to read numbers.

**Comment by nebmo** [comment]
Yes, same here. But what do colors do you associate with speed up/slow down?

**Comment by cmmata** [comment]
In my opinion, your initial colors where ok:
- Green should be on target.
- Red usually is for alert, so I should use it for too slow (you need to speed up) like you said.
- Blue usually is for information, so it's a good choice for too fast since it's not as bad as too slow.

What do you think?

**Comment by nebmo** [comment]
I will try it out! 
Also a question about configuration: For intervals when you set duration and target i guess that information is the most important to show on watch. Maybe i should add a config setting that , if checked, will override any other other configured values for middle and top text views. 

Example: In configuration for interval i have set:
top: distance
middle: pace:
bottom: none
Then I start a interval with target= 160 HR, and duration = 2 min. then these settings will be overridden and HR and time is displayed instead of distance and pace.

What do you think?

**Comment by cmmata** [comment]
That would be great! Much more useful than showing always pace and distance. Also, if bottom line is not used, it could help if you show target value there. That way, you have colors to show you're going too fast / too slow, and a numeric value to see how much.

**Comment by jonasoreland** [comment]
should it not be like this ??

Example: In configuration for interval i have set:
top: duration
middle: target value
bottom: none

then there will be no "override"...
but same behaviour

**Comment by nebmo** [comment]
Yes Jonas. but what if interval does not have target set? :) what should be displayed then?
Right now i use Dimension and Scope to configure what to show. Then I have to use something else to define it

**Comment by gerhardol** [comment]
Some improvements done in 1.58
